civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -5579,9 +5579,10 @@ function _civihr_employee_portal_form_view_exposed_form_alter_people_report(&$fo
  * @param string $form_id
  */
 function _civihr_employee_portal_form_view_exposed_form_alter_leave_report(&$form, $form_state, $form_id) {
-  $key = 'absence_date_filter';
-  $form[$key]['min'] = setInputDatepickerForm($form[$key]['min']['#title']);
```

Comment:
So, before this change, the field label was set according to this `#title` property, right? Why did you have to manually change this by passing a different title to `setInputDatepickerForm()` instead of changing the value this `#title` property?

Reply:
We talked about this and, since this report solution will soon be removed, the changes in this PR are ok